theano/tensor/tests/test_extra_ops.py

```diff
@@ -424,6 +424,7 @@ def test_repeatOp(self):
                     r_var = T.scalar(dtype=dtype)
                     r = numpy.asarray(3, dtype=dtype)
                     if dtype in self.numpy_unsupported_dtypes:
+                        r_var = T.vector(dtype=dtype)
```

Comment:
If the error is  not raised for a `T.scalar`, then we should test that the value is right, like in the `else:` block.